main.tf

```
@@ -40,6 +40,42 @@ resource "aws_kms_external_key" "this" {
   tags = var.tags
 }
 
+################################################################################
+# Replica Key
+################################################################################
+
+resource "aws_kms_replica_key" "this" {
+  count = var.create && var.create_replica && !var.create_external && !var.create_replica_external ? 1 : 0
+
+  bypass_policy_lockout_safety_check = var.bypass_policy_lockout_safety_check
+  deletion_window_in_days            = var.deletion_window_in_days
+  description                        = var.description
+  primary_key_arn                    = var.primary_key_arn
+  enabled                            = var.is_enabled
+  policy                             = coalesce(var.policy, data.aws_iam_policy_document.this[0].json)
+
+  tags = var.tags
+}
+
+################################################################################
+# Replica External Key
+################################################################################
+
+resource "aws_kms_replica_external_key" "this" {
+  count = var.create && !var.create_replica && !var.create_external && var.create_replica_external ? 1 : 0
```

likewise here as well, can we just do
```suggestion
  count = var.create && var.create_replica_external ? 1 : 0
```

---

disregard